tf2onnx/tfonnx.py

```diff
@@ -766,6 +766,14 @@ def concat_op(ctx, node, name, args):
 def concatv2_op(ctx, node, name, args):
     # T output = ConcatV2(T values, Tidx axis, @int N, @type Tidx)
     # T concat_result = Concat(T inputs, @INT axis)
+    # if any input is empty, remove this concat op
+    # TODO: file a issue to onnxruntime that empty input of concat op
+    # will leads seg fault
+    for i, inp in enumerate(node.inputs):
+        if inp.is_const() and inp.get_tensor_value() == []:
+            ctx.replace_all_inputs(ctx.get_nodes(), node.output[0], node.inputs[1-i].output[0])
+            return
```

Comment:
you will only handle on input, return is used correctly?

Reply:
concat might have more than two inputs, what if two are valid, the other one is []?

Reply:
orz, thanks!